thirdparty/jquery-entwine/dist/jquery.entwine-dist.js

```diff
@@ -1465,23 +1436,6 @@ catch (e) {
 
 (function($) {	
 
-	/** Taken from jQuery 1.5.2 for backwards compatibility */
-	if ($.support.changeBubbles == undefined) {
-		$.support.changeBubbles = true;
-
-		var el = document.createElement("div");
-		eventName = "onchange";
-
-		if (el.attachEvent) {
-			var isSupported = (eventName in el);
-			if (!isSupported) {
-				el.setAttribute(eventName, "return;");
-				isSupported = typeof el[eventName] === "function";
-			}
-
-			$.support.changeBubbles = isSupported;
-		}
-	}
```

Comment:
`$.support` is deprecated - and anyway we aren't interested in supporting such old browsers so the easy fix to the errors this caused was to remove it.